Examples/test-suite/common.mk

```diff
@@ -966,6 +966,7 @@ swig_and_compile_c =  \
 
 swig_and_compile_multi_cpp = \
 	for f in `cat $(top_srcdir)/$(EXAMPLES)/$(TEST_SUITE)/$*.list` ; do \
+	  f=`printf $${f} | grep -v '^$$'`; \
```

Comment:
The behave of the shell in Windows.
'f' variable contain the windows new line at the end of it.
Then it can not find the "${f}.i" files when calling 'swig_and_compile_cpp_helper' function.
In Linux, this do not change 'f' value. So it is should work on all  platforms.